docs/cpp/config-clang-mac.md

```diff
@@ -110,10 +110,25 @@ Remember, the C++ extension uses the C++ compiler you have installed on your mac
 
    ![Screenshot of helloworld.cpp and play button](images/playbutton/run-play-button.png)
 
-3. Choose **C/C++: clang++ build active file** from the list of detected compilers on your system (you'll only be asked to choose a compiler the first time you run `helloworld.cpp`).
+3. Choose **C/C++: clang++ build and debug active file** from the list of detected compilers on your system.
 
    ![Build task](images/clang-mac/default-build-task.png)
 
+you'll only be asked to choose a compiler the first time you run `helloworld.cpp`. This compiler will be set as the "default" compiler in `tasks.json` file. In case you need to change the default compiler, you can run **Tasks: Configure default build task**. Alternatively you can modify the `tasks.json` file and remove the default by replacing this segment:
+
+```json
+    "group": {
+        "kind": "build",
+        "isDefault": true
```

Comment:
There is also a mention of `isDefault` lower in the section. Do you think we need both? 
Would it be better to move this section down to replace the other?
For the tutorial flow, readers are introduced to `tasks.json` a couple of paragraphs below and after that seems a better time to mention how to change the "default" compiler.

Reply:
I agree.